README.md

```diff
@@ -141,14 +141,15 @@ WBS Deploy and WBS Images are released using this repository. The process involv
 ```
 - [ ] **Pending issues as subtasks**. If any open tickets need to be resolved and/or related changes need to be included in the release, add them as subtasks of this release ticket.
 - [ ] **To release breaking changes** as a new major version of WBS Deploy, create a new branch called `deploy-X`, where `X` is the new major version.
-- [ ] **Create a release PR** with the following changes targeting the appropriate `deploy-X` release branch.
+- [ ] **Create a release PR** from a release preparation branch with the following changes targeting the appropriate `deploy-X` release branch.
+  - [ ] **Backports from `main`** obtained by cherrypicking commits commits from `main` to the release preparation branch.
```

Comment:
wait, I understand the sentence but there is sth weird I can't put my finger on. Is there a typo? "commits" is written twice but also the rest seems off. I may be the one off, thought :) Can an english native check this, please?

Reply:
Indeed, good catch, check out the update please.

>   - [ ] **Backport from `main`** by cherrypicking commits from `main` to the release preparation branch.